src/utils/api.ts

```
@@ -0,0 +1,281 @@
+  return isNetworkError(error) || isServerError(error);
+};
+
+/**
+ * Retry an async function with exponential backoff
+ */
+export const retryWithBackoff = async <T>(
+  fn: () => Promise<T>,
+  maxRetries: number = 3,
+  baseDelay: number = 1000
+): Promise<T> => {
+  let lastError: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)